src/lib/rule-finder.js

```diff
@@ -72,13 +72,17 @@ function _getPluginRules(config) {
 function _getAllAvailableRules(pluginRules) {
   var allRules = fs
     .readdirSync('./node_modules/eslint/lib/rules')
-    .map(function removeJsFromFilename(filename) {
-      return filename.replace(/\.js$/, '')
-    })
+  var filteredAllRules = []
+  allRules.forEach(function filterAllRules(filename) {
+    /* istanbul ignore next */
```

Comment:
since `eslint` is mocked, I have [updated the mock's return value](https://github.com/sarbbottam/eslint-find-rules/pull/133/files#diff-912d04f91de422a081173c41ff31335fR10).